### ISSUE TITLE
`[Processing: package_name]` cannot be disabled via `--event-handlers` toggles

### ISSUE THREAD

**Issue by roncapat** [colcon/colcon-parallel-executor#37]
I noticed that prints like `[Processing: package_name]` cannot be filtered out.
Is there a way to do so?

**Comment by cottsay** [comment]
I don't believe there's any way to do that right now. It's a hardcoded print:
https://github.com/colcon/colcon-parallel-executor/blob/4157ad934ed60b78e5eba32de9307771ae8a6450/colcon_parallel_executor/executor/parallel.py#L165-L167

We could probably route it through the event handling system so that it could be controlled.
